code/game/objects/items/storage/pouch.dm

```diff
@@ -593,9 +592,26 @@
 	else
 		return ..()
 
-/obj/item/storage/pouch/flare/full
-	fill_type = /obj/item/explosive/grenade/flare
-	fill_number = 7
+/obj/item/storage/pouch/flare/attackby_alternate(obj/item/I, mob/user, params)
+	if(!istype(I, /obj/item/weapon/gun/grenade_launcher/single_shot/flare))
+		return ..()
+	var/obj/item/weapon/gun/grenade_launcher/single_shot/flare/flare_gun = I
+	for(var/obj/item/flare in contents)
+		if(!(flare.type in can_hold))
+			continue
+		if(user.l_hand && user.r_hand || length(flare_gun.chamber_items))
+			flare_gun.tactical_reload(flare, user)
```

Comment:
tactical reloads only work with magazines. Does this actually work, and is this just copied from pistol code?

Reply:
Admittedly, yeah, I pasted this over from pistol code and didn't really bother giving it much of a read since it was working when I tested it. I can notice that this is unneeded, and I just tested it, so I'll go ahead and fix that bit of code.